### PR TITLE
ChripCSVDBService handles the limit of cheeps read

### PR DESCRIPTION
**Pull request by Maquidos**
The functionallity of the limit was before in Chirp.CLI.Client But is now in ChirpCSVDBService. The Read methode is responsable for limiting the amount of cheeps returned fro the http get call. The http call is now sending success or fail responses back, which is then displayed to the user.

### CODE REVIEW

File: src/Chirp.CLI.Client/Program.cs [hunk 1]
```diff
@@ -63,48 +63,36 @@ static async Task Main(string[] args)
         // gets the commands from the commandline and exectutes the following code
         readCommand.SetHandler(async (limitOptionValue) =>
         {
-            if (limitOptionValue < 0)
-            {
-                System.Console.WriteLine("the --limit option should be greater than or equal to 0");
-                return;
-            }
+            // if (limitOptionValue < 0)
```
Comment (RazzerTazzer) [inline src/Chirp.CLI.Client/Program.cs:66]: Delete this, if it not nescessary anymore

File: src/Chirp.CSVDBService/Program.cs [hunk 2]
```diff
@@ -16,23 +32,31 @@
...
+                records = all_records;
+            }
+            return records;
         }
     }
     else
     {
-        Console.WriteLine($"There are no cheeps to read from here: {path}\nPlease specify a different path, or simply begin cheeping to create some cheeps!");
-        return null;
+        throw new FileNotFoundException($"There are no cheeps to read from here: {path}\nPlease specify a different path, or simply begin cheeping to create some cheeps!");
```
Comment (palleBrandt) [inline src/Chirp.CSVDBService/Program.cs:59]: Do we still need the part of "specify new path"?

File: src/Chirp.CLI.Client/Program.cs [hunk 3]
```diff
@@ -63,48 +63,36 @@ static async Task Main(string[] args)
...
-            if (limitOptionValue < 0)
-            {
-                System.Console.WriteLine("the --limit option should be greater than or equal to 0");
-                return;
-            }
+            // if (limitOptionValue < 0)
+            // {
+            //     System.Console.WriteLine("the --limit option should be greater than or equal to 0");
+            //     return;
+            // }
```
Comment (palleBrandt) [inline src/Chirp.CLI.Client/Program.cs:70]: Delete these lines

**Review by palleBrandt — CHANGES_REQUESTED** [review]
Very good. 

Would like the outcommented lines to be deleted. #Cleanup

**Review by RazzerTazzer — APPROVED** [review]
Loooks good man

File: src/Chirp.CSVDBService/Program.cs (hunk 2 above)
Comment (Maquidos) [reply src/Chirp.CSVDBService/Program.cs:59]: It is for the sake of the developer, to indicate that the path is not found to the CSV file

Comment (palleBrandt) [reply src/Chirp.CSVDBService/Program.cs:59]: Makes sence, keep it.

File: src/Chirp.CLI.Client/Program.cs [hunk 4]
```diff
@@ -63,48 +63,36 @@ static async Task Main(string[] args)
...
             
-            // IEnumerable<Cheep> records = data_access.Read(limitOptionValue);
-            
-            //The records are passed to the UserInterface, which handles how the records are presented to the user
-            List<Cheep> all_records;
-            List<Cheep> relevant_records = new List<Cheep>();
+            List<Cheep> records;
             string baseURL = "http://localhost:5089/";
```
Comment (RazzerTazzer) [inline src/Chirp.CLI.Client/Program.cs:73]: Change the base url to azure?

**Review by palleBrandt — APPROVED** [review]
Looks good

**Pull request merged**